### PR TITLE
create cart feature

### PR DESCRIPTION
**Pull request by abo389**

### CODE REVIEW

File: app/Http/Controllers/CartController.php
Comment (Huma-volve) [inline app/Http/Controllers/CartController.php:1]: 1-Also here use the trait to return response to frontend in all your controller

File: app/Http/Requests/CartRequest.php
Comment (Huma-volve) [inline app/Http/Requests/CartRequest.php:1]: always make your request like this in array 
            "product_id" => ['required']

File: app/Models/Cart.php
Comment (Huma-volve) [inline app/Models/Cart.php:1]: Why making product_id, user_id in hidden and also updated_at, and created_at

**Review by ghost — CHANGES_REQUESTED** [review]

File: app/Models/Cart.php
Comment (abo389) [reply app/Models/Cart.php:1]: because front end doesn't need this data

**Pull request merged**